v3/library-after-success.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+set -e
+
+# Necessary to stop pull requests from forks from running.
+if [[ "$TRAVIS_SECURE_ENV_VARS" == "true" && -n "$TRAVIS_TAG" ]]; then
+  # Allow package.json to specify a custom build script.
+  if npm run | grep -q build:ci; then
+    echo -e "Running custom build step... `npm run build:ci`";
+    npm run build:ci
```

Comment:
@Blackbaud-BobbyEarl Some of our libraries need to do a few other actions after building the library (and before pushing to NPM). This condition checks if the package.json contains a script for `npm run build:ci` and runs that, instead of `skyux build-public-library`.

I've tested this on the alpha release of `@skyux/config`, here:
https://github.com/blackbaud/skyux-config/blob/master/package.json#L12

The library `@skyux/theme` will also require a custom build step.